repeat_expansion_variants/clinvar_identifier_parsing.py

```diff
@@ -0,0 +1,108 @@
+"""
+The regular expressions in this module are used to parse ClinVar's identifiers for describing repeat expansion variants.
+The pyhgvs module cannot be applied here because not all expression used by ClinVar are actually valid HGVS, and that
+module imposes strict validation. Hence, custom regular expressions are necessary.
+"""
+
+import re
+
+
+# Common part for all HGVS-like transcript definitions, e.g. 'NM_001256054.2(C9orf72):'
+hgvs_like_transcript_part = (
+    r'(?P<transcript_id>[A-Za-z0-9_]+)'  # Transcript accession                      NM_001256054
+    r'\.'                                # Delimiter, transcript accession/version   .
+    r'[0-9]+'                            # Transcript version                        2
+    r'[A-Za-z0-9_.()]*'                  # Gene symbol in parentheses, optional      (C9orf72)
```

Comment:
I guess it's probably fine (because of the surrounding version number and colon) but this regex will also match things like `1a(2b)3c`. Consider forcing the parenthesis at the start/end only, if that's the actual grammar, although this looks horrible:

```suggestion
    r'(?:\([A-Za-z0-9_.]*\))?'           # Gene symbol in parentheses, optional      (C9orf72)
```

Reply:
Fair point! Your assumption is right, it is the actual grammar. I was just lazy when writing this part of a regexp. I have implemented your suggestions, splitting into multiple lines for readability:
https://github.com/EBIvariation/vep-mapping-pipeline/pull/3/commits/290dbbca998b7db2a4d60d05cce255b9b5d8c3f3